alica_engine/include/engine/IAlicaCommunication.h

```diff
@@ -46,7 +46,7 @@ class IAlicaCommunication
     virtual void sendSolverResult(const SolverResult& sr) const = 0;
     virtual void sendAgentQuery(const AgentQuery& pq) const = 0;
     virtual void sendAgentAnnouncement(const AgentAnnouncement& pa) const = 0;
-    virtual void sendLogMessage(int /*level*/, const std::string& /*message*/) const {};
+    virtual void sendLogMessage([[maybe_unused]] int level, [[maybe_unused]] const std::string& message) const {};
```

Comment:
Why changed here? It was already fixed here

Reply:
I accidentally did similar changes based on `0.9.4`.
Kept for consistency, and so the compiler is more involved in checking.

Reply:
My concern is it became longer, readability worse IMHO
And in other places I used same approach with /**/ in other patch.

Reply:
Hm, wonder if/when this will be applied to `0.9.4`.
Maybe it's better to just fix the `0.9.4.` directly instead, for now.
@Dale-Koenig any suggestions?

Reply:
@xffox Sure, I think it's reasonable for you to make the changes on 0.9.4 first as long as they are not too large.

Reply:
Moved here https://github.com/rapyuta-robotics/alica/pull/368 .